src/backends/web.jl

```diff
@@ -33,11 +33,15 @@ function open_browser_window(filename::AbstractString)
 end
 
 function write_temp_html(plt::AbstractPlot)
-    html = standalone_html(plt; title = plt.attr[:window_title])
     filename = string(tempname(), ".html")
-    output = open(filename, "w")
-    write(output, html)
-    close(output)
+    if plt.backend == PlotlyJSBackend()
```

Comment:
Can you instead define `function write_temp_html(plt::Plot{PlotlyJSBackend}) ...` in the backend file?